steps/wip/measure.py

```diff
@@ -0,0 +1,290 @@
+import json
+from typing import Dict, Optional, Union
+
+from zquantum.core.bitstring_distribution import save_bitstring_distribution
+from zquantum.core.circuit import (
+    Circuit,
+    load_circuit,
+    load_circuit_connectivity,
+    load_circuit_set,
+    load_circuit_template_params,
+)
+from zquantum.core.hamiltonian import (
+    estimate_nmeas_for_frames,
+    estimate_nmeas_for_operator,
+    get_expectation_values_from_rdms,
+    get_expectation_values_from_rdms_for_qubitoperator_list,
+)
+from zquantum.core.measurement import load_expectation_values, save_expectation_values
+from zquantum.core.openfermion import (
+    load_interaction_rdm,
+    load_qubit_operator,
+    load_qubit_operator_set,
+)
+from zquantum.core.typing import Specs
+from zquantum.core.utils import (
+    create_object,
+    load_noise_model,
+    save_list,
+    save_nmeas_estimate,
+    save_value_estimate,
+)
+from zquantum.core.wip.estimators.estimation import (
+    estimate_expectation_values_by_averaging,
+)
+
+
+def run_circuit_and_measure(
```

Comment:
It doesn't look to me like this function needs to be in wip. Same for many others.

Reply:
I thought it would just be easier to replace the whole file since many of the imports changed

Reply:
I'd say it's riskier, as your changes affect more code.
And harder to review, cause it's not obvious what has actually changed ;)

Reply:
Not trying to fight you on this one, but genuinely curious how you would then distinguish between removing a piece of code in a wip module because it is unaffected vs it no longer being needed.

Reply:
That's a very good point :D